xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1080,20 +1080,21 @@ bool CGUIWindowVideoBase::OnPlayMedia(int iItem, const std::string &player)
   CServiceBroker::GetPlaylistPlayer().Reset();
   CServiceBroker::GetPlaylistPlayer().SetCurrentPlaylist(PLAYLIST::TYPE_NONE);
 
-  CFileItem item(*pItem);
+  CFileItemPtr pItemCopy = std::make_shared<CFileItem>(*pItem);
```

Comment:
I think there is a `Clone()` method available.

Also, no `p` prefixes.

Reply:
yes there is but is not needed on make_shared CFileItem constructor that want the CFileItem reference